### PR TITLE
Completed data-structures-linked-lists exercise - Noah Lett

### PR DESCRIPTION
**Pull request by NoahLett**
image

### CODE REVIEW

File: data-structures-linked-lists/append.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported append */
+
+function append(list, value) {
+  list.next.next.next.next = new LinkedList(value);
```
Comment (Zhiwutian) [inline data-structures-linked-lists/append.js:5]: you want to update this function to work with any size list

File: data-structures-linked-lists/insert-next.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported insertNext */
+
+function insertNext(list, value) {
+  list.next = new LinkedList(value, list.next.data, list.next.next.data, list.next.next.next.data);
```
Comment (Zhiwutian) [inline data-structures-linked-lists/insert-next.js:5]: Same as above, this should work with any size list

File: data-structures-linked-lists/prepend.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported prepend */
+
+function prepend(list, value) {
+  const prependList = new LinkedList(value, list.data, list.next.data, list.next.next.data, list.next.next.next.data);
```
Comment (Zhiwutian) [inline data-structures-linked-lists/prepend.js:5]: as before, make sure the function works for all sized lists

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Noah, just make sure to update the functions noted so that they would work with any sized list.

**Comment by NoahLett** [comment]
image
image
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**